core/iwasm/fast-jit/jit_ir.c

```diff
@@ -1558,3 +1558,43 @@ _jit_insn_check_opnd_access_LookupSwitch(const JitInsn *insn)
     unsigned opcode = insn->opcode;
     return (insn_opnd_kind[opcode] == JIT_OPND_KIND_LookupSwitch);
 }
+
+#if defined(BUILD_TARGET_X86_64) || defined(BUILD_TARGET_AMD_64)
+bool
+jit_lock_regs_in_insn(JitCompContext *cc, JitInsn *the_insn, JitReg locked_reg)
```

Comment:
If we only lock one reg, had better change the name to `jit_lock_reg_in_insn`.
If to lock multiple regs, had better change the prototype to:
```C
bool
jit_lock_regs_in_insn(JitCompContext *cc, JitInsn *the_insn, JitReg *regs_to_lock, uint32 reg_count)
```

And had better change `locked_reg` to `reg_to_lock`